sourcing/module.go

```diff
@@ -23,12 +26,13 @@ func (m *Module) Inject(
 	config *struct {
 		UseDefaultSourcingService bool `inject:"config:commerce.sourcing.useDefaultSourcingService,optional"`
 		EnableQtyRestrictor       bool `inject:"config:commerce.sourcing.enableQtyRestrictor,optional"`
+		UseFakeService            bool `inject:"config:commerce.sourcing.fake.enable, optional"`
```

Comment:
Please mention the new config value and the needed json structure in the readme of the module.